eitprocessing/eit_data/__init__.py

```diff
@@ -109,10 +109,12 @@ def from_path(
                 else:
                     eit, continuous, sparse = loaded_data
 
-                # assertions for type checking
-                assert isinstance(eit, DataCollection)  # noqa: S101
-                assert isinstance(continuous, DataCollection)  # noqa: S101
-                assert isinstance(sparse, DataCollection)  # noqa: S101
+                if TYPE_CHECKING:
+                    # These asserts tells the type checker they can assume these are the types of these variables.
+                    # Required due to multiple return types for _from_path().
+                    assert isinstance(eit, DataCollection)
+                    assert isinstance(continuous, DataCollection)
+                    assert isinstance(sparse, DataCollection)
```

Comment:
If it's just for type checking, then I believe you can also do this:

```suggestion
                eit: DataCollection
                continuous: DataCollection
                sparse: DataCollection
```

Reply:
Fixed this in 695c2abb1b72d0fe57f5a35b531fdaa9968c3a4f.